omod/src/main/resources/sqldiff.xml

```diff
@@ -98,5 +98,17 @@
 		</sql>
 	</diff>
 	
-        
+        <diff>
+		<version>1.3</version>
+		<author>Choxmi Sathsara</author>
+		<date>June 26th 2017</date>
+		<description>
+			Added 'defaultreportquery' and 'mappeddefinitionlabel' to datavalue_template table
+		</description>

```

Comment:
Going forwards, we do not recommend the use of sqldiff.xml
Can you use liquibase.xml?